decko-rails/lib/decko/engine.rb

```diff
@@ -27,7 +26,7 @@ def gem_root
 
   end
 
-  class Decko::Engine < Rails::Engine
+  class Engine < ::Rails::Engine
     
     paths.add "app/controllers", :eager_load => true
     paths.add 'gem-assets',      :with => 'public/assets'
```

Comment:
By "never got clear on", I'm not so much talking about what you wrote, but what I've seen when I tried in and looked into how it is supposed to work.  As with most rails and ruby things, the code is better than the documentation, and lots of examples and descriptions are at best partly right.

Is the Decko::Engine.paths object the same or different from the MyApp::Application.paths object?  Same question for config.  I think they are necessarily different, but I'm also not sure that is right.